python/ray/tune/trial_runner.py

```diff
@@ -229,13 +229,14 @@ def step(self):
         """
         if self.is_finished():
             raise TuneError("Called step when all trials finished?")
-        self.trial_executor.on_step_begin()
-        next_trial = self._get_next_trial()
+        with warn_if_slow("on_step_begin"):
```

Comment:
Would I see these warnings in the application logs in between the tune results?

Reply:
Yeah